modules/core/shared/src/main/scala/io/circe/ACursor.scala

```diff
@@ -222,6 +222,15 @@ abstract class ACursor(private val lastCursor: HCursor, private val lastOp: Curs
    */
   def downField(k: String): ACursor
 
+  /**
+   * Navigate down through multiple fields in a JSON object sequentially.
+   *
+   * @param ks The sequence of field names to navigate through.
+   * @return A cursor positioned at the final field if all specified fields exist,
+   *         or a failed cursor if any field in the sequence cannot be found.
+   */
+  def downFields(ks: String*): ACursor
```

Comment:
`downFields` should require at least one, this way you don't have to do the is empty check later.
```suggestion
  def downFields(k: String, ks: String*): ACursor
```